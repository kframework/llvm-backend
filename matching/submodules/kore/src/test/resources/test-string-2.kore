[key{}("\a")]

module TEST-STRING-2

endmodule []
