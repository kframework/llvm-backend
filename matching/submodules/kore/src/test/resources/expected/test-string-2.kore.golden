Definition
    { definitionAttributes =
        Attributes 
            [ Fix (UnifiedPattern (UnifiedObject (Rotate31 (ApplicationPattern Application
                { applicationSymbolOrAlias =
                    SymbolOrAlias
                        { symbolOrAliasConstructor = (Id "key" AstLocationNone) :: Id Object
                        , symbolOrAliasParams = []
                        }
                , applicationChildren =
                    [ Fix (UnifiedPattern (UnifiedMeta (Rotate31 (StringLiteralPattern (StringLiteral "\a")))))
                    ]
                }))))
            ]
    , definitionModules =
        [ Module
            { moduleName = ModuleName "TEST-STRING-2"
            , moduleSentences = []
            , moduleAttributes = Attributes []
            }
        ]
    }